### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by el119** [McJty/RFTools#214]
It would be great if there was something like a RF powered Terraformer (like IC2) in this mod, but better. 
The terraformer should have something like Dimlets, that specify what Biome type or what Liquids and Blocks there sould be.

**Comment by McJty** [comment]
Yes, something like that has already crossed my mind. For later though :-)
